Don't require C11.
This requirement arose from a bug in Harfbuzz 2 which was fixed in 2.3.0.
Reverts https://github.com/kovidgoyal/kitty/commit/f098e6c973c5b514edb6047546bdd90c0705332b
--- setup.py.orig	2019-01-03 22:07:59.000000000 -0600
+++ setup.py	2019-01-14 20:26:50.000000000 -0600
@@ -190,7 +190,7 @@
     cppflags = shlex.split(cppflags)
     cflags = os.environ.get(
         'OVERRIDE_CFLAGS', (
-            '-Wextra -Wno-missing-field-initializers -Wall -std=c11'
+            '-Wextra -Wno-missing-field-initializers -Wall -std=c99'
             ' -pedantic-errors -Werror {} {} -fwrapv {} {} -pipe {} -fvisibility=hidden'
         ).format(
             optimize,
